fighthealthinsurance/rest_urls.py

```diff
@@ -1,17 +1,34 @@
+from django.urls import include
 from django.urls import path
+from django.conf import settings
 
 from fighthealthinsurance import rest_views
 
-rest_urls = [
+from rest_framework import routers
+
+if settings.DEBUG:
```

Comment:
This lets us see an API index at the root of the `/figgy/rest/` endpoint in development mode (or whenever **DEBUG** is **True**) but it won't show the legacy APIs since the'yre not viewsets.